### PR TITLE
Updating to RTM builds of xunit

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @dougbu

File: test/Microsoft.Extensions.DiagnosticAdapter.Test/project.json [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
   "dependencies": {
-    "dotnet-test-xunit": "1.0.0-*",
+    "dotnet-test-xunit": "2.2.0-*",
     "Microsoft.AspNetCore.Testing": "1.1.0-*",
     "Microsoft.Extensions.DiagnosticAdapter": "1.1.0-*",
-    "xunit": "2.1.0-*"
-  },
-  "buildOptions": {
-    "define": [
-      "TEST"
```
Comment (dougbu) [inline test/Microsoft.Extensions.DiagnosticAdapter.Test/project.json]: Why isn't this define needed? Or, why was it ever needed?

**Comment by dougbu** [comment]
One question but :shipit: if the `TEST` define is just leftovers.

File: test/Microsoft.Extensions.DiagnosticAdapter.Test/project.json (hunk 1 above)
Comment (pranavkm) [inline test/Microsoft.Extensions.DiagnosticAdapter.Test/project.json]: Doesn't look like it's used anywhere. @rynowak any guesses what this is about?

**Pull request merged**